README.md

```diff
@@ -41,9 +50,8 @@ Before we start, make sure you fit the prerequisites, the source code is inside
 - `npm run deploy`
 
 ## Authors
```

Comment:
- [ ] Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.